README.md

```diff
@@ -594,7 +594,8 @@ Running `lerna` without arguments will show all commands/options.
       "ignore": "component-*"
     }
   },
-  "packages": ["packages/*"]
+  "packages": ["packages/*"],
+  "publishDir": "lib/"
```

Comment:
Would it be worth changing this to `dist/` or `build/` it feels like a more common use case for developers